test/e2e/scenarios/cmd_basic_create_update.yaml

```diff
@@ -0,0 +1,40 @@
+id: "RG_CMD_basic_create_update"
+description: "In this test we create RG with absolutely minimum configuration and try to update it."
+usecases:
+  - replication_group
+  - cmd
+#marks:
+#  - slow
+#  - blocked
+customResourceReference:
+  apiVersion: $CRD_GROUP/$CRD_VERSION
+  kind: ReplicationGroup
+  metadata:
+    name: test$RANDOM_SUFFIX
```

Comment:
I would strongly urge you to rename this to `namePrefix` instead of name and assume that every metadata `name` is suffixed. Any tests that run in parallel would otherwise try to create resources with the same name and break soak tests.

Reply:
`RANDOM_SUFFIX` is added to the `repacements` before loading the scenario.
Each scenario gets a different value for it, thus parallel runs do not cause same name.

Reply:
I understand that, but that requires developers adding `$RANDOM_SUFFIX` at the end of every `metadata.name`. I think we still need `$RANDOM_SUFFIX` for when developers want to create a random resource name (such as randomly name their cluster), but for `metadata.name` I think we should take control. Another suggestion is that we just don't allow adding `metadata` in these scenario files at all, and we use the ID of the scenario as the name? Further, it would be great to run each set of scenarios in a separate namespace.

Reply:
Any comments on this @kumargauravsharma ? I still feel it leaves room for mistakes as it currently exists. If everyone is just going to have to append $RANDOM_SUFFIX anyway, then lets just do it for them?